app/src/main/java/org/oppia/android/app/drawer/DeveloperOptionsStarterModule.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.android.app.drawer
+
+import dagger.Binds
+import dagger.Module
+
+@Module
+abstract class DeveloperOptionsStarterModule {
```

Comment:
What is the purpose of this module?

Reply:
Since `@BindsOptionalOf` does not take any parameters, I had to create this module to provide dependencies for the starter implementation. You can find the reference [here](https://stackoverflow.com/a/43007805)